### ISSUE TITLE
Transition duration should not be shared across all transitions

### ISSUE THREAD

**Issue by Zhuinden** [Zhuinden/simple-stack-compose-integration#10]
The `animationSpec` should be provided by `CustomComposableTransitions`.

Once `animationSpec` is merged into`CustomComposableTransitions`, it will be shown that `CustomComposableTransitions` is the same thing as `AnimationConfiguration`.

Therefore, in a future 0.x.0 version, animation duration must be detached to allow configurability for specific screen combinations, and not be shared across all screen changes.

**Comment by Zhuinden** [comment]
Fixed in https://github.com/Zhuinden/simple-stack-compose-integration/commit/d586e2f7389802312f32277f7626911c368e20da released in 0.5.0
